explorer/ui/app/entry.server.tsx

```diff
@@ -11,9 +11,21 @@ import { createReadableStreamFromReadable } from '@remix-run/node'
 import { RemixServer } from '@remix-run/react'
 import { isbot } from 'isbot'
 import { renderToPipeableStream } from 'react-dom/server'
+import { Counter, collectDefaultMetrics } from 'prom-client'
 
 const ABORT_DELAY = 5_000
 
+let reqsCount: any = null
```

Comment:
maybe avoid `any`

Reply:
Yeah avoid any. Also avoid let if we can - can thro stuff below into a function that returns `Counter`.

Also - not sure we need to allow for this to be `null`? We should be able to have metrics code always runs. What changes per env is - what `env` the metric is tagged with, and whether or not those metrics are scraped .

Would rather not have to do null checks every time we want to set a metric.